bindings/go/fdb-go-install.sh

```diff
@@ -126,6 +127,12 @@ function parseArgs() {
         shift
     done
 
+    if [[ -z "${FDBVER}" ]] ; then
+        echo "No FoundationDB version specified!"
+        echo "Please provide a value to the --fdbver or set the FDBVER environment variable."
```

Comment:
Should this remove the 'the' before `--fdbver` or add a noun (e.g. 'argument') after `--fdbver`?

Reply:
I rewrote this in a way that I think is better.